src/lib/components/modal.js

```diff
@@ -103,24 +103,6 @@ function registerPostMessageCallback(completionCallback) {
       } else {
         redirect(event.options);
       }
-    } else if (event.action == 'height' && event.type == 'modal') {
```

Comment:
so we never want the iframe to get its height adjusted, but instead rely on scrolling?